.travis.yml

```diff
@@ -31,7 +31,7 @@ install:
 script:
   - export PYTHONPATH=`pwd`:$PYTHONPATH
   - python src/tests/runtests.py
-  # - nunit-console src/embed_tests/bin/x64/ReleaseMono/Python.EmbeddingTest.dll
+  # - nunit-console src/embed_tests/bin/Python.EmbeddingTest.dll
```

Comment:
@filmor if we downgrade back to nunit `2.6.2` we can start running the tests. They will still fail, but nunit will gracefully fail and give us the error instead of whatever nunit3 is doing now